### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by vic778**
In this project, 
- I implement the CRUD (create, update, delete) methods. All the elements of the user interface will be fully functional and my application is completed.
- to delete an element you should select it first.

### CODE REVIEW

**Comment by vic778** [comment]
# I deploy it on netlify because my github forces me to create another pull request that is not allowed in this project. but the god part of the project I will deploy it on github because it will contain a new pull request

File: src/addToList.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default (stock, todo) => {
+  stock.push({
+    description: todo,
+    completed: 0,
+    index: stock.length + 1,
+  });
+  localStorage.setItem('stock', JSON.stringify(stock));
+};
```
Comment (Dipeshtwis) [inline src/addToList.js]: - [ ] Suppose we have two tasks already and when we refresh the page and then try adding a new task all previous tasks got removed from the localstorage. Please fix this issue.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @vic778, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: You have made a separate module for each feature. Well Done.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addToList.js [hunk 2]
```diff
@@ -5,4 +5,4 @@ export default (stock, todo) => {
     index: stock.length + 1,
   });
   localStorage.setItem('stock', JSON.stringify(stock));
-};
\ No newline at end of file
+};
```
Comment (bimbolabuari) [inline src/addToList.js]: As requested by the previous code reviewer 🎉 
> Suppose we have two tasks already and when we refresh the page and then try adding a new task all previous tasks got removed from the local storage. Please fix this issue.

Kindly fix this issue 🙂

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @vic778, 

You have done a good job so far 🚀  

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Kindly fix the remove implementation as the delete icon is deleting the wrong object (task). 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Victoir, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**